webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -180,6 +181,8 @@ const NavigationSideBar: React.FC = () => {
     openWorkspaceCreateModal();
   }, [openWorkspaceCreateModal]);
 
+  const pinnedWorkspaces = Loadable.getOrElse([], useWorkspaces({ limit: 0, pinned: true }));
```

Comment:
If you're not going to handle the loading states in this PR can you leave TODOs where they need to be handled?